src/ComparedQueryable/ComparedQueryable.nuspec

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Buildium.ComparedQueryable</id>
+    <version>0.1.0</version>

```

Comment:
I made this first version of the `ComparedQueryable` project version 0.1.0 to indicate that it was in a very beta state.